src/Uno.UI/UI/Xaml/FrameworkElement.EffectiveViewport.cs

```diff
@@ -375,7 +375,7 @@ internal ViewportInfo GetParentViewport()
 				_effectiveViewportChanged?.Invoke(this, new EffectiveViewportChangedEventArgs(parentViewport.Effective));
 			}
 
-			if (_childrenInterestedInViewportUpdates is { Count: > 0 } && (isInitial || viewportUpdated))
+			if (_childrenInterestedInViewportUpdates is { Count: > 0 } && (isInitial || viewportUpdated || _lastScrollOffsets != ScrollOffsets))
```

Comment:
Ok makes sense 👍

Reply:
Indeed you changed the `GetEffectiveViewport` accordingly ... but only on skia. You will probably either have to change also on other platform, either have to make this new condition platform specific.

Reply:
I guess this new condition will always match viewportUpdated on the other platforms? I would prefer keeping it (redundantly) instead of adding `#if __SKIA__`.

Reply:
Indeed ... a small comment would be great then :)